example/default/json_serializable_span.dart

```diff
@@ -14,29 +14,48 @@
 
 import 'package:opentracing/opentracing.dart';
 
+/// Abstract class that identifies other classes being serializable as json.
+///
+/// Implementing this class will enable a class to be serialized using the
+/// JSON.encode method.
 class JsonSerializableSpan {
+  // ignore: public_member_api_docs
```

Comment:
BTW you can ignore this lint for the whole file (https://www.dartlang.org/guides/language/analysis-options#ignoring-specific-analysis-rules)

`// ignore_for_file: public_member_api_docs`

Reply:
ty